Documentation/DocuBlocks/Rest/Administration/get_admin_log.md

```diff
@@ -218,176 +218,179 @@ the API is disabled, all requests will be responded to with HTTP 403. If the
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{audit-view,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
 
 @RESTBODYPARAM{authentication,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{authorization,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{backup,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{bench,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{cache,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{cluster,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{clustercomm,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
-@RESTBODYPARAM{collector,string,optional,string}
-One of the possible log levels.
+@RESTBODYPARAM{collector,string,optional,string} !
+One of the possible log topics.
 
 @RESTBODYPARAM{communication,string,optional,string}
-One of the possible log levels.
-
-@RESTBODYPARAM{compactor,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{config,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{crash,string,optional,string}
-One of the possible log levels.
-
-@RESTBODYPARAM{datafiles,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{development,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{dump,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{engines,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{flush,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{general,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{graphs,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{heartbeat,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{httpclient,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{ldap,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
+
+@RESTBODYPARAM{libiresearch,string,optional,string}
```

Comment:
not 100% sure this should be present here

Reply:
File system-related ArangoSearch problems are logged under this topic and we suggested to adjust the level e.g. in https://github.com/arangodb/arangodb/issues/12628#issuecomment-697927571

Reply:
ok!